styles/base.css

```diff
@@ -10,10 +10,10 @@ body {
 }
 
 body {
-  @apply scroll-smooth
-    bg-white
+  @apply bg-white
     font-open-sans
     text-neutral-950
+    motion-safe:scroll-smooth
```

Comment:
is it still need ? 'cause we make scrool smooth on layout.

Reply:
I'm not sure if there is a place in the body where scroll behavior is used, it would be best to get other contributors comments and consider removing it 🤔 

Since I wasn't sure if it was used somewhere, I added motion-safe and left it as is 😄